### DISCUSSION TITLE
Welcome to jvppeteer Discussions!

### DISCUSSION THREAD

**Discussion by fanyong920** [fanyong920/jvppeteer#39] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ty1360114681** [comment]
你好，我在使用过程中遇到一个问题，在win环境下可以手动安装内核到.local-browser目录下能被寻找并正常启动，但是到linux环境下此操作就会异常提示Could not find browser revision 722234. Pleaze download a browser binary.
